scripts/lib/CIME/get_timing.py

```diff
@@ -29,20 +29,13 @@ def __init__(self, case, lid="999999-999999"):
         self.adays=0
         self._driver = case.get_value("COMP_INTERFACE")
         self.models = {}
+        self.ncount = 0
+        self.nprocs = 0
```

Comment:
This file is a mess and really needs to be completely rewritten.   Please spot check the resulting timing files and make sure that the numbers seem reasonable to you.